org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WProcessParameter.java

```diff
@@ -54,59 +57,76 @@ public ADForm getForm() {
 				for(MProcessPara processPara : processParameters) {
 					if (processPara.getColumnName().equals(para.getParameterName())) {
-						schedulerPara.setAD_Process_Para_ID(processPara.getAD_Process_Para_ID());
+						poPara.set_ValueOfColumn("AD_Process_Para_ID", processPara.getAD_Process_Para_ID());
 						if (DisplayType.isNumeric(processPara.getAD_Reference_ID())) {
 							if (para.get_Value(MPInstancePara.COLUMNNAME_P_Number) != null)
-								schedulerPara.setParameterDefault(para.getP_Number().toString());
+								poPara.set_ValueOfColumn("ParameterDefault", para.getP_Number().toString());
 							if (processPara.isRange() && para.get_Value(MPInstancePara.COLUMNNAME_P_Number_To) != null) 
-								schedulerPara.setParameterToDefault(para.getP_Number_To().toString());
+								poPara.set_ValueOfColumn("ParameterToDefault", para.getP_Number_To().toString());
 						} else if (DisplayType.isID(processPara.getAD_Reference_ID())) {
 							if (para.get_Value(MPInstancePara.COLUMNNAME_P_Number) != null)
-								schedulerPara.setParameterDefault(Integer.toString(para.getP_Number().intValue()));
+								poPara.set_ValueOfColumn("ParameterDefault", Integer.toString(para.getP_Number().intValue()));
 							if (processPara.isRange() && para.get_Value(MPInstancePara.COLUMNNAME_P_Number_To) != null)
-								schedulerPara.setParameterToDefault(Integer.toString(para.getP_Number_To().intValue()));
+								poPara.set_ValueOfColumn("ParameterToDefault", Integer.toString(para.getP_Number_To().intValue()));
 						} else if (DisplayType.isDate(processPara.getAD_Reference_ID())) {
 							if (para.getP_Date() != null)
-								schedulerPara.setParameterDefault(para.getP_Date().toString());
+								poPara.set_ValueOfColumn("ParameterDefault", para.getP_Date().toString());
 							if (processPara.isRange() && para.getP_Date_To() != null)
-								schedulerPara.setParameterToDefault(para.getP_Date_To().toString());
+								poPara.set_ValueOfColumn("ParameterToDefault", para.getP_Date_To().toString());
 						} else {
-							schedulerPara.setParameterDefault(para.getP_String());
+							poPara.set_ValueOfColumn("ParameterDefault", para.getP_String());
 							if (processPara.isRange() && para.getP_String_To() != null)
-								schedulerPara.setParameterToDefault(para.getP_String_To());
+								poPara.set_ValueOfColumn("ParameterToDefault", para.getP_String_To());
 						}
 						if (!Util.isEmpty(para.getInfo())) {
-							schedulerPara.setDescription(para.getInfo());
+							poPara.set_ValueOfColumn("Description", para.getInfo());
 						}
 						if (!Util.isEmpty(para.getInfo_To())) {
-							String s = schedulerPara.getDescription();
+							String s = poPara.get_ValueAsString("Description");
 							if (Util.isEmpty(s))
 								s = para.getInfo_To();
 							else
 								s = s + ", " + para.getInfo_To();
-							schedulerPara.setDescription(s);
+							poPara.set_ValueOfColumn("Description", s);
```

Comment:
I'm not sure how do you mean this, since poPara can be an instance of MSchedulerPara or MProcessDrillRulePara here. 
Perhaps there could be checked, if the given column exists?